static/src/stylesheets/module/content/_content.scss

```diff
@@ -154,6 +154,22 @@
     }
 }
 
+// Styling for long section/series labels at the leftCol breakpoint
+.content__labels--long {
+    @include mq(leftCol, wide) {
+        .content__section-label__link {
+            @include fs-header(2);
+            line-height: 22px;
+            display: block;
+            padding: $gs-baseline/6 0;
+        }
+        .content__series-label__link {
+            @include fs-headline(2);
+            display: block;
```

Comment:
Since they're inline elements there seems to be a cap on the minimum line-height relative to the font-size, therefore I had to include the display:block. Is there another way to solve this?

![gif](https://cloud.githubusercontent.com/assets/2236852/5725716/51db54fa-9b4d-11e4-9ffb-fcb2f9079487.gif)